railties/lib/rails/application/configuration.rb

```diff
@@ -205,6 +205,7 @@ def load_defaults(target_version)
           if respond_to?(:action_view)
             action_view.button_to_generates_button_tag = true
             action_view.apply_stylesheet_media_default = false
+            action_view.frozen_string_literal = true
```

Comment:
We need to document this on `configuring.md`

Reply:
Damn it, I keep forgetting that one. Done now.